### PR TITLE
Login screen

### PR DESCRIPTION
**Pull request by SandraMarcelaHerreraArriaga**

### CODE REVIEW

File: src/components/ButtonFirebase/ButtonFirebase.js [hunk 1]
```diff
@@ -30,7 +30,32 @@ function ButtonFirebase(){
...
+                    grant_type: 'firebase',
+                    firebase_token_id: idToken
+                  }
+                },{
+                  headers: {
+                    "Content-type":"application/x-www-form-urlencoded",
+                  }
+                }, {
+                  auth: {
+                    username: 'iOSApp',
```
Comment (iuresti) [inline src/components/ButtonFirebase/ButtonFirebase.js]: how could we avoid publishing these sensitive information to the repository?

File: src/components/NavBar/NavBar.css [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
     background-color:rgb(94, 93, 156);    
     color: #ffffff;    
-    padding: 10px;     
+    padding: 10px; 
+    
 }
 .logo-navbar{
     background-color: lawngreen;
-    width: 250px;
+    width: 500px;
```
Comment (iuresti) [inline src/components/NavBar/NavBar.css]: fixed sizes usually are not the best options .. can you use `em`s or percentages ?

File: src/config.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export const config = {
```
Comment (iuresti) [inline src/config.js]: Do not publish this to the repository

File: src/components/Footer/index.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import './Footer.css'
+
+function Footer(){
+    return <div className="footer-div">Powered by UASLP-DESARROLLADOR SLP- NEARSOFT</div>
+}
+export default Footer
```
Comment (hectorbenitez) [inline src/components/Footer/index.js]: Add a newline at end of each file

File: src/components/ButtonFirebase/index.js [hunk 5]
```diff
@@ -24,6 +24,36 @@ const sendToBackendToken = () => {
   })
 }
 
+const firebaseAuth = () => () => {
+  firebase.auth().currentUser.getIdToken(/* forceRefresh */ true).then(function (idToken) {
```
Comment (hectorbenitez) [inline src/components/ButtonFirebase/index.js]: This code, can be part of a service in other file

Comment (SandraMarcelaHerreraArriaga) [reply src/components/ButtonFirebase/index.js]: Thanks for the code review Hector, I changed it :)

File: src/components/NavBar/NavBar.css (hunk 2 above)
Comment (SandraMarcelaHerreraArriaga) [reply src/components/NavBar/NavBar.css]: Thanks for code review Ivan I changed it :)

**Pull request merged**